java-example/auto-instrumentation/README.MD

```diff
@@ -0,0 +1,93 @@
+# Auto Instrumenting Java Apps with OpenTelemetry
```

Comment:
@keep94 @oppegard 
This is the new content from the blog. Did some minor formatting change. I will test it out too within this week